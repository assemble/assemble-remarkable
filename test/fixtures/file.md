# Foo

> This is a blockquote

This is a paragraph

