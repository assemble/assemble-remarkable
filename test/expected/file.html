<h1>Foo</h1>
<blockquote>
<p>This is a blockquote</p>
</blockquote>
<p>This is a paragraph</p>
